### PR TITLE
Migration: Update docs/framework/install/guide-for-developers

### PR DESCRIPTION
**Pull request by guardrex**

### CODE REVIEW

**Comment by GuardRex** [comment]
@rpetrusha ready for review+merge.

File: docs/framework/install/guide-for-developers.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ manager: "wpickett"
 The fact that later .NET Framework 4.x versions are in-place updates to earlier versions means that you *cannot* install an earlier version listed in the table if a later version is already installed. For example, you cannot install the .NET Framework 4.6 on a Windows 10 November Update system, since the .NET Framework 4.6.1 is preinstalled on it.    
   
 > [!NOTE]
->  For information on the .NET Framework 3.5, see [Installing the .NET Framework 3.5 on Windows 8 and later versions](../Topic/Installing%20the%20.NET%20Framework%203.5%20on%20Windows%208,%20Windows%208.1%20and%20Windows%2010.md).  
+>  For information on the .NET Framework 3.5, see [Installing the .NET Framework 3.5 on Windows 8 and later versions](~/docs/framework/install/dotnet-35-windows-10.md).
```
Comment (rpetrusha) [inline docs/framework/install/guide-for-developers.md]: The link you've provided is a user installation topic, not a developer installation topic. The correct link is [Installing the .NET Framework 3.5 on Windows 8, Windows 8.1 and Windows 10](net-framework-3-5-on-windows-8-plus.md).

File: docs/framework/install/guide-for-developers.md [hunk 2]
```diff
@@ -201,12 +201,12 @@ The fact that later .NET Framework 4.x versions are in-place updates to earlier
...
   
 ## See Also  
  [Deployment Guide for Developers](~/docs/framework/deployment/deployment-guide-for-developers.md)   
  [Deployment Guide for Administrators](~/docs/framework/deployment/guide-for-administrators.md)   
- [Installing the .NET Framework 3.5 on Windows 8 and later versions](../Topic/Installing%20the%20.NET%20Framework%203.5%20on%20Windows%208,%20Windows%208.1%20and%20Windows%2010.md)   
- [Troubleshooting](~/docs/framework/install/troubleshoot-blocked-installations-and-uninstallations.md)
\ No newline at end of file
+ [Installing the .NET Framework 3.5 on Windows 8 and later versions](~/docs/framework/install/dotnet-35-windows-10.md)
```
Comment (rpetrusha) [inline docs/framework/install/guide-for-developers.md]: Again, the wrong topic. The correct link is above.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Again, some issues with link targets.

**Review by rpetrusha — APPROVED** [review]
LGTM.

**Pull request merged**